manager/runner.go

```diff
@@ -27,9 +27,9 @@ import (
 )
 
 const (
-	// saneViewLimit is the number of views that we consider "sane" before we
```

Comment:
IMO both of these variable names are to long. What do you think of simply `viewLimit`?